.github/workflows/pylint.yaml

```diff
@@ -18,12 +18,12 @@ jobs:
       matrix:
         os:
           - name: fedora
-            version: 29
-            python: 2
+            version: 31
+            python: 3
```

Comment:
We still need to run pylint in python 2. IMO it does not make much sense to run it in both supported version of fedora for python 3.

It seems to be available in epel7 though, through this package: https://koji.fedoraproject.org/koji/buildinfo?buildID=1403259

We could run it in CentOS7 with EPEL for python 2. Thoughts?

Reply:
Cent makes more sense than Fedora, for sure.

Reply:
@athos-ribeiro; Alrighty then. I'm assuming that we were stuck on F29 x Py2 because "No package python-pylint available" on Cent7 x Py2.

Reply:
We need to install it from EPEL, which is not automatically enabled in CentOS (you have to install the epel package). This was the reason we did not go with it a while ago. But now we do not have python 2 in Fedora anymore, it makes sense to go this way.

Reply:
@athos-ribeiro; Latest checks show that I've won the battle but lost the war. EPEL's pylint (v1.6.5 - the only one available; indeed it's the same for Py3) is either too stupid to recognize our pylint config, or too stupid to lint *only* the delta in this PR. Either way, it returns about 10 million hits (i.e. instead of 0)  ¯\\\_(ツ)_/¯

Reply:
As we're dropping Py2 support as soon as humanly possible, we've decided to stick with F29 for Py2 pylint but note that it's essentially deprecated, and open an issue to drop Py2 pylint when possible.